### DISCUSSION TITLE
Benefits not received & Bugs and advice in applying process

### DISCUSSION THREAD

**Discussion by Protomyst** [community/community#145391] [GitHub Education]
### Hi everyone,

# Issues with Onboarding Process

It has been 3 days since my approval, but I still haven't received the benefits. I understand they might become available after a few days, but I wonder if providing feedback here could speed up the process.

There were also some bugs and my advice during my application process and my advice:

- When I took a picture of my academic proof using a mobile device, I could only use the front-facing camera, regardless of which option I selected.
- I believe the option to upload a file should be made available again.
- The location detection is not accurate enough (since it's inferred from IP address, there's some margin of error). In my case, it was off by 100 km, which led to me being identified as not being near the school.

**Comment by nawaftahir** [comment]
I had also applied and haven't received my benefits as a student.

**Comment by Protomyst** [comment]
![image](https://github.com/user-attachments/assets/73fbafa9-9776-4c45-9d77-df0f6a1f70cb)
Now received. Perhaps the feedback works?

**Reply by nawaftahir** [reply]
Maybe I'll have to make my own separate happy you got yours!

**Reply by Protomyst** [reply]
Thanks~

**Reply by SachinVardhan06** [reply]
Bro I also applied and my application got approved but not received benifits 1 day gone @Protomyst

**Reply by Protomyst** [reply]
I use **3 days**, so be patient

**Reply by SachinVardhan06** [reply]
Ok bro 🫡

Edit :- Bro u try to contact GitHub support of anything else...

**Comment by nevzyalo** [comment]
also confirmed 2 days ago, but did not issue benefits

**Reply by Protomyst** [reply]
The time of benefit receipt is unpredictable
